scripts/sync_charts.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# No unset vars, print commands as they're executed, and exit on any non-zero
+# return code
+set -u
+set -x
+set -e
+
+if [[ "$#" -ne 1 ]]; then
+    echo "Usage: sync_charts.sh <istio/installer branch-name>"
```

Comment:
I don't mean to be super negative...  This appears to be a dev tool to sync the charts into the repo.  I am not sure what the advantage of baked in charts is.  I can think of numerous disadvantages.

Can you elaborate on the advantages?

Reply:
regardless of whether they are baked in or not, they can still be sourced from istio/installer, and they will be by end of this week, per my other comment. 
the baked in stuff will be removed now that we have a release process for these charts defined. once that's implemented and working, the baked in stuff will be removed.